lib/tasks/locale.rake

```diff
@@ -135,6 +135,10 @@ namespace :locale do
 
   desc "Extract plugin strings - execute as: rake locale:plugin:find[plugin_name]"
   task "plugin:find", :engine do |_, args|
+    unless args.key?(:engine)
+      puts "You need to specify a plugin name: rake locale:plugin:find[plugin_name]"
+      next
```

Comment:
Keep in mind rake can handle multiple rake tasks in line... Like `rake db:migrate db:rollback`.  `next` may allow the next task to run.  Maybe?  I don't know.  Is that what we want here?